Examples/Python/src/Geometry.cpp

```diff
@@ -19,12 +20,40 @@
 namespace py = pybind11;
 using namespace pybind11::literals;
 
+namespace {
+struct GeometryIdentifierHookBinding : public Acts::GeometryIdentifierHook {
+  py::object callable;
+
+  Acts::GeometryIdentifier decorateIdentifier(
+      Acts::GeometryIdentifier identifier,
+      const Acts::Surface& surface) const override {
+    return callable(identifier, surface.getSharedPtr())
+        .cast<Acts::GeometryIdentifier>();
+  }
+};
+}  // namespace
+
 namespace Acts::Python {
 void addGeometry(Context& ctx) {
   auto m = ctx.get("main");
-
-  py::class_<Acts::Surface, std::shared_ptr<Acts::Surface>>(m, "Surfaces");
-
+  {
+    py::class_<Acts::Surface, std::shared_ptr<Acts::Surface>>(m, "Surfaces")
+        .def("volumeId",
+             [](Acts::Surface& self) { return self.geometryId().volume(); })
+        .def("boundaryId",
+             [](Acts::Surface& self) { return self.geometryId().boundary(); })
+        .def("layerId",
+             [](Acts::Surface& self) { return self.geometryId().layer(); })
+        .def("approachId",
+             [](Acts::Surface& self) { return self.geometryId().approach(); })
+        .def("sensitiveId",
+             [](Acts::Surface& self) { return self.geometryId().sensitive(); })
+        .def("extraId",
+             [](Acts::Surface& self) { return self.geometryId().extra(); })
+        .def("center", [](Acts::Surface& self) {
+          return self.center(Acts::GeometryContext{});
+        });
+  }
   {
```

Comment:
Would it make sense to just add bindings to `GeometryIdentifier` itself, and add a binding for the `geometryId()` getter itself?

Reply:
Ah, `GeometryIdentifier` has bindings already, so do we need these extra methods on `Surface`?

Reply:
actually not I would say, I'll remove them in favor of a geometry identifier accessor.